llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8956,8 +8965,17 @@ LoopVectorizationPlanner::tryToBuildVPlanWithVPRecipes(VFRange &Range) {
       // with the final reduction value will be added to the exit block
```

Comment:
Update comment, possibly merge with the new one below?

Reply:
76d4dde1b0a9a7a6c0184180ad02f77db23c7447
And replace `last` with `final` for aligning the term with the comment of `isInvariantStoreOfReduction`.
```
  /// Returns True if given store is a final invariant store of one of the
  /// reductions found in the loop.
  bool isInvariantStoreOfReduction(StoreInst *SI);
```